styles/_base.scss

```diff
@@ -209,8 +268,8 @@ ul, ol {
 
   #chatConvoContainer {
     position: fixed;
-    right: 0;
-    bottom: 42px;
+    right: $scrollbar;
+    bottom: 1px;
```

Comment:
The designs have the bottom border of the chat conversation being visible, so I bumped it up a pixel.